frontend/src/routes/(authenticated)/admin/ProjectTable.svelte

```diff
@@ -72,6 +74,18 @@ async function softDeleteProject(project: Project): Promise<void> {
         notifyWarning($t('delete_project_modal.success', {name: project.name, code: project.code}));
     }
 }
+
+  const migrationStatusTable = {
+    [ProjectMigrationStatus.Migrated]: 'i-mdi-checkbox-marked-circle-outline',
+    [ProjectMigrationStatus.Migrating]: 'loading loading-spinner loading-xs',
```

Comment:
I like this. How hard would it be to have the spinner on the project-page too?

Reply:
I'd have to loosen up the type restrictions on the Icon component to allow IconStrings that don't start with `i-mdi`, but then I could do it.

Reply:
Perhaps `IconStrings` should be extended to included exactly `loading loading-spinner loading-${string}` 😆. Strange, but perhaps appropriate.

Reply:
Done.